roles/fio-distributed/templates/client.yaml

```diff
@@ -51,7 +47,7 @@ spec:
 {% for numjobs in fiod.numjobs %}
 {% for bs in fiod.bs %}
 {% for job in fiod.jobs %}
-             cat /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}}; mkdir -p /tmp/fiod-{{ uuid }}/fiojob-{{job}}-{{bs}}-{{numjobs}}; python /opt/snafu/run_snafu.py -t fio -H /tmp/host/hosts -j /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}} -s {{fiod.samples}} -d /tmp/fiod-{{ uuid }}/fiojob-{{job}}-{{bs}}-{{numjobs}};
+             cat /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}}; mkdir -p /tmp/fiod-${uuid}/fiojob-{{job}}-{{bs}}-{{numjobs}}; python /opt/snafu/run_snafu.py -t fio -H /tmp/host/hosts -j /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}} -s {{fiod.samples}} -d /tmp/fiod-${uuid}/fiojob-{{job}}-{{bs}}-{{numjobs}};
```

Comment:
Why the change from {{ uuid }} to ${uuid}? It only seems to be done in fio and uperf